tests/us_reverse_geo/test_Result.js

```diff
@@ -0,0 +1,41 @@
+const chai = require("chai");
+const expect = chai.expect;
+const SmartyResponse = require("../../src/us_reverse_geo/SmartyResponse");
```

Comment:
I kinda commented on this above, but why are we using `SmartyResponse` instead of the `Result` class? This test is named `test_Result` so something isn't named correctly for consistency and `SmartyResponse` is never used in the production code